registry.json

```diff
@@ -1,4 +1,19 @@
 [
+   {
+      "ticker":"RSV",
+      "poolId":"pool1r0sxz0hygfc0266lhdclve6j3cwcwlth6dya6uey65yxjnwm7qc",
+      "social":{
+         "facebook":"",
+         "twitter":"@RSVNODE",
+         "website":"https://reservoir.network",
+         "reddit":"",
+         "instagram":"",
+         "youtube":""
+      },
+      "telegram":"@reservoirNode",
+      "operatorName":"contact",
```

Comment:
If you don't provide your Real name, please leave operatorName empty.

Reply:
Hello, I'm new to github. Hope I didnt screw this up. thanks for understanding